client/idp/src/app/manager/homepage/homepage.component.css

```diff
@@ -0,0 +1,70 @@
+.bg-opacity {
+    opacity: 80%;
+    color: black
+}
+
+.container-fluid {
+    background-color: rgb(246, 247, 249);
+    padding: 2rem;
+
+}
+
+.text-left {
+    font-weight: bolder;
+    color: black;
+    position: relative;
+    top: 0 em;
+    left: 05 em;
+}
+
+h2 {
+    font-size: 36px;
+    text-align: center;
+    margin-top: 20px;
+}
+
+h6 {
```

Comment:
Why not use the bootstrap `text-center` class instead?

Reply:
It has changed into bootstrap.